extensions/amp-analytics/0.1/analytics-root.js

```diff
@@ -339,6 +346,29 @@ export class AnalyticsRoot {
    * @abstract
    */
   createVisibilityManager() {}
+
+  /**
+   *  Returns the Scroll Managet corresponding to this analytics root.
+   * The Scroll Manager is created lazily as needed, and will handle
+   * calling all handlers for a scroll event.
+   * @return {!./scroll-manager.ScrollManager}
+   */
+  getScrollManager() {
+    if (!this.scrollManager_) {
+      this.scrollManager_ = this.createScrollManager();
```

Comment:
Since the `getScrollManager` won't be override by sub class. You don't need a `createSrollManager()` method. 
`this.scrollManager_ = new ScrollManager(this.ampdoc);` would help saving few lines of code : )